app/services/data/get-pending-tasks-and-mark-in-progress.js

```diff
@@ -0,0 +1,35 @@
+const config = require('../../../knexfile').development
+const knex = require('knex')(config)
+const taskStatus = require('../../constants/task-status')
+const Task = require('../domain/task')
+
+module.exports = function (schema, batchSize) {
```

Comment:
Is there any way we could split this out to two separate actions? I know Andy was looking to make it more granular and move any manipulation/behaviour to the services and domain so that data just solely retrieved data. But this may be a bad usecase, I am not sure enough about the workflow yet.